.github/workflows/scenario-test.yaml

```
@@ -1,106 +1,95 @@
 name: scenario-test
 on:
+  schedule:
+    - cron: 0 12 * * *
   workflow_dispatch:
-
+  pull_request:
+    paths:
+      - simulator*.repos
+      - .github/workflows/scenario-test.yaml
 jobs:
   scenario-test:
-    runs-on: [self-hosted, Linux, X64]
-    container: ghcr.io/autowarefoundation/autoware:universe-devel
+    runs-on: ubuntu-24.04
+    container:
+      image: ghcr.io/autowarefoundation/autoware:universe-devel
     steps:
       - name: Check out repository
         uses: actions/checkout@v4
-
-      - name: Show disk space before the tasks
-        run: df -h
-
-      - name: Show machine specs
-        run: lscpu && free -h
-
       - name: Install required packages
+        shell: bash
         run: |
           apt-get update
           apt-get install -y \
             unzip \
+            zip \
             curl \
             gnupg2 \
             lsb-release \
             python3-pip \
             git \
             ros-humble-pcl-ros
```

You shouldn't have to install ros-humble-pcl-ros

```suggestion
            git
```